velox/exec/tests/SharedArbitratorTest.cpp

```diff
@@ -1072,6 +1076,7 @@ TEST_F(SharedArbitrationTest, concurrentArbitration) {
     const int maxNumZombieTasks = 8;
     std::vector<std::thread> queryThreads;
     queryThreads.reserve(numThreads);
+    TestScopedAbortInjection testScopedAbortInjection(10, numThreads);
```

Comment:
We don't have to limit the max injections? Maybe consider to have one run without abort? Thanks!